src/core/test_helpers/kbn_server.ts

```diff
@@ -118,6 +100,15 @@ export function createRoot(settings = {}, cliArgs: Partial<CliArgs> = {}) {
  *  @returns {Root}
  */
 export function createRootWithCorePlugins(settings = {}, cliArgs: Partial<CliArgs> = {}) {
+  const DEFAULT_SETTINGS_WITH_CORE_PLUGINS = {
```

Comment:
Well, it's a hack to prevent failure when the file is imported in the unit-test environment, since `esTestConfig.getUrl` relies on `TEST_ES_URL` env var. 
@spalger is there a way to add a guard check we are in `integration tests` or `functional tests` env?